### ISSUE TITLE
cloud-launch doesn't panic into cloud logs

### ISSUE THREAD

**Issue by josephholsten** [camlistore/camlistore#737]
I'm doing an initial import into a cloud-launched instance. The app is clearly restarting every so often, but I'm getting zero debugging output from the instance. The only indication that the service is restarting is

`"2016/04/28 05:06:46 Starting camlistored version 7b78c50007; Go go1.5.2 (linux/amd64)`

**Comment by mpl** [comment]
You're talking about the "Custom logs" category in the Google Cloud Logging entries for your project, right?

Well there wouldn't be much to see anyway. If camlistored is dying, you may see an error message about why it's dying (and if not, we should fix that). But then it's dead, and the systemd service would be the one restarting it.

Maybe you'll get a tiny bit more info with journalctl? From your server instance page, ssh into the instance and look at the logs with 'sudo journalctl -u camlistored'.

**Comment by josephholsten** [comment]
Yeah, I'm not seeing the error that's causing the process to die.

I'll check to see if anything's going into journald.
